portable/NetworkInterface/board_family/ReadMe.txt

```diff
@@ -1 +1,5 @@
-Update NetworkInterface.c and include other files needed by FreeRTOS+TCP here.
\ No newline at end of file
+Update NetworkInterface.c and include other files needed by FreeRTOS+TCP here.
+
+Note: This particular NetworkInterface.c is Not! to be used as is. The purpose of this file is to provide a template for writing a network interface.
+Each network interface will have to provide concrete implemtations of the functions in this file.
+See the following URL for an explanation of this file and its functions: https://freertos.org/FreeRTOS-Plus/FreeRTOS_Plus_TCP/Embedded_Ethernet_Porting.html
```

Comment:
Same comment. This is a readme. Maybe this should be reworded.

Reply:
Updated readme.